src/state/opamUpdate.ml

```diff
@@ -199,7 +199,7 @@ let pinned_package st ?version ?(working_dir=false) name =
     let version =
       OpamFile.OPAM.version_opt opam ++
       version +!
-      OpamPackage.Version.of_string "dev"
+      OpamPackage.Version.default
```

Comment:
oh weird, I thought i had them all changed in #4949. My bad.

Reply:
:)